sdk/identity/azure-identity/inc/azure/identity/azure_cli_credential.hpp

```diff
@@ -50,10 +57,15 @@ namespace Azure { namespace Identity {
    * token.
    */
   class AzureCliCredential
-#if !defined(TESTING_BUILD)
+#if !defined(_azure_TESTING_BUILD)
```

Comment:
Because doxygen has a bug since it fails to generate the docs for types that use macros that have the `final` keyword in them. Our `PrePublishBuild GenerateReleaseArtifacts` steps fails with a doxygen warning treated as error :(

So, if we use a macro, the classes that use it, don't show up in the generated docs.

https://github.com/doxygen/doxygen/issues/10641